src/Volt.php

```diff
@@ -9,6 +9,7 @@
  * @method static void mount(array|string $paths = [], array|string $uses = [])
  * @method static array paths()
  * @method static \Livewire\Features\SupportTesting\Testable test(string $name, array $params = [])
+ * @method static self withQueryParams(array $params = [])
```

Comment:
That "self" does not seem right?

Reply:
On VSCODE it works correctly, because it returns Volt facade itself. So we can chain another methods.